doc/deployment_guide_s390.md

```diff
@@ -0,0 +1,147 @@
+# Running D-Installer on a z/VM machine
```

Comment:
I would use "Agama". #509 is already merged, so "D-Installer" is not used anymore in the rest of the repo.